ui/src/app/components/target-details-form/target-details-form.component.html

```diff
@@ -1,6 +1,9 @@
 <div mat-dialog-content>
   <form [formGroup]="targetDetailsForm" class="target-detail-form">
-    <h2>Spanner Database Details</h2>
+    <h2>Spanner Database Details<mat-icon class="configure"
+      matTooltip='Schema & Schema and data migration: Specify new database name or existing database with empty schema. Data migration: Specify existing database with tables already created'>
```

Comment:
Is there a toolTip available on the prepare-migration-page's configure database button itself? That seems like a more suitable place to add this.